### PR TITLE
Variables doc updates for V8

### PR DESCRIPTION
**Pull request by SueSmith**
Pending anything I have wrong about the request access bits...

### CODE REVIEW

File: src/pages/docs/sending-requests/variables.md [hunk 1]
```diff
@@ -141,41 +141,36 @@ Enter a **Name**, verify the **Value** and select a scope from the drop-down lis
...
 
-You can add collection variables when you create the collection or at any time after that. To create or edit a variable for an existing collection, select the collection in __Collections__ on the left of the Postman app, open the __View more actions__ (__...__) menu, and click __Edit__.
-
-<img alt="Edit Collection" src="https://assets.postman.com/postman-docs/edit-collection-var.jpg" width="350px"/>
-
-> If you only have __Viewer__ role on the collection, you will see the __View Details__ menu item instead of __Edit__ (on the collection and any folders inside it).
-> ![View collection details](https://assets.postman.com/postman-docs/view-details-collection-viewer.jpg)
-
-Choose the __Variables__ tab to edit or add to your collection variables.
+You can add collection variables when you create the collection or at any time after that. To create or edit a variable for an existing collection, select the collection in __Collections__ on the left of the Postman app and choose **Variables**.
```
Comment (sowmyaraj92) [inline src/pages/docs/sending-requests/variables.md:166]: Remove 'app'.

File: src/pages/docs/sending-requests/variables.md [hunk 2]
```diff
@@ -141,41 +141,36 @@ Enter a **Name**, verify the **Value** and select a scope from the drop-down lis
...
 
-![Collection Variables](https://assets.postman.com/postman-docs/collection-vars.jpg)
+[![Edit Collection](https://assets.postman.com/postman-docs/collection-vars-v8.jpg)](https://assets.postman.com/postman-docs/collection-vars-v8.jpg)
```
Comment (sowmyaraj92) [inline src/pages/docs/sending-requests/variables.md:168]: Can we resize the image for better readability?

File: src/pages/docs/sending-requests/variables.md [hunk 3]
```diff
@@ -184,7 +179,7 @@ You can also [define collection variables in scripts](#defining-variables-in-scr
 
 You can add and edit variables at any time. All you need to include for a new variable is a name—you can choose to supply an initial value but can alternatively set it later, including from scripts. Use the checkbox to enable or disable a variable.
 
-![Manage Variables](https://assets.postman.com/postman-docs/manage-vars.jpg)
+[![Edit Collection](https://assets.postman.com/postman-docs/collection-vars-v8.jpg)](https://assets.postman.com/postman-docs/collection-vars-v8.jpg)
```
Comment (sowmyaraj92) [inline src/pages/docs/sending-requests/variables.md:182]: Image resize here too.

**Review by sowmyaraj92 — COMMENTED** [review]
@SueSmith LGTM 🥳 Just a few changes and we are ready to go 🚀

**Comment by SueSmith** [comment]
@sowmyaraj92 I don't think I'm going to have time to do the edits on all of these so if you can just go ahead and make any changes you think are required that would be magic 🙏

**Review by sowmyaraj92 — APPROVED** [review]

**Pull request merged**